src/styles/partials/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,73 @@
+.dg_breadcrumbs {
+  color: $black;
+  display: inline-block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  list-style: none;
+  margin-bottom: 0px;
+  margin-top: $margin-small;
+  padding-bottom: $padding-smallest;
+  padding-left: 0px;
+  padding-top: 0px;
+  text-transform: uppercase;
+
+  i {
+    font-size: $h3-font-size;
+  }
+
+  ul {
+    margin-bottom: 0px;
+    padding-bottom: 0px;
+  }
+
+  li {
+    display: inline;
+
+    a {
+      color: $blue;
+      font-size: $smallest-font-size;
+      font-weight: $font-normal;
+      padding: 0px 30px 0px 0px;
+      text-decoration: none;
+
+      &:hover {
+        color: $vibrant-blue;
+      }
+
+      &:after {
+        color: $gray-light;
+        font-size: $h1-font-size;
+        font-weight: $font-bold;
+        @include icon-content("\f105");
+        padding-left: 30px;
+        vertical-align: top;
```

Comment:
i believe this should be middle, please take a look how that looks, and let me know.

Reply:
This appears to work with a middle align as well. I have switched it to vertical-align: middle; and pushed it up.